pkg/controller/bookkeepercluster/bookie.go

```diff
@@ -280,6 +301,16 @@ func createVolumeMount(ledgerDirs []string, journalDirs []string, indexDirs []st
 		MountPath: heapDumpDir,
 	}
 	volumeMounts = append(volumeMounts, v)
+	if len(extraVolumeMounts) > 1 {
+		for _, vm := range extraVolumeMounts {
+			s := strings.Split(vm, "=")
+			v := corev1.VolumeMount{
+				Name:      s[0],
+				MountPath: s[1],
```

Comment:
Do we use the same path for both mountPath and hostPath?

Reply:
Yes, we use the same path for both.